### DISCUSSION TITLE
convert mkv files to mp4 before moving to ./done?

### DISCUSSION THREAD

**Discussion by ralyodio** [transmission/transmission#2031] [Q&A] [answered]
I'm using transimission daemon on Ubuntu and I'm wondering if there's a way with ffmpeg to convert a `.mkv` file to `.mp4` in same directory as the `.mkv` file before moving to the ./done folder.

**Comment by ile6695** [comment, marked as answer]
Converting files is not the main thing for torrent clients.
`transmission-qt` has setting for calling a script when completing a torrent, but you might want something that works on daemon.

Daemon has an option for moving completed files to another folder on completion.
You could set the daemon to move the files to some other folder first, have another script watching that folder using `inotifywait` and then have that script convert and move the new files to ./done
